doc/topics/windows/windows-package-manager.rst

```diff
@@ -1137,22 +1112,21 @@ cache_file (str)
 ----------------
 
 This setting requires the file to be stored on the ``file_roots`` and only
-applies to URLs that begin with ``salt://``. It indicates a single file to copy
-down for use with the installer. It is copied to the same location as the
-installer. Use this over ``cache_dir`` if there are many files in the directory
-and you only need a specific file and don't want to cache additional files that
-may reside in the installer directory.
+applies to URLs that begin with ``salt://``. It indicates that only a single file specified
+is  to be copied down for use with the installer. It is copied to the same location as the
```

Comment:
**Nitpick:** Maybe remove "to be" in this sentence.

Reply:
Updated